class Solution {
    public String removeOuterParentheses(String s) {
        String str="";
        int count=0;
        for(int i=0;i<s.length();i++){
            char ch=s.charAt(i);
            if(ch=='('){
                if(count>0){
             str=str+ch;
                }
                count++;
            }else{
                count--;
                if(count>0){
                    str=str+ch;
                }
            }
        }
        return str;
    }
}